src/components/layouts/pages/inspect/tree/Tree.js

```diff
@@ -63,6 +74,9 @@ function Tree({ ruleSummaries, selectedCheck, setSelectedCheck, errorsMap }) {
     // info dialog props
     const [openedRule, setOpenedRule] = useState(UNSELECTED);
     const [infoDialogOpened, setInfoDialogOpened] = useState(false);
+    const errorMessages = useMemo(() => {
+        return errorMessagesMap[profile === errorProfiles.TAGGED_PDF ? profile : errorProfiles.OTHER][language];
```

Comment:
Let's use switch to get the profile name in order to make this logic more flexible